src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/dates/ReleaseDateService.kt

```diff
@@ -89,31 +89,26 @@ class ReleaseDateService(
   @TimeServedConsiderations("For time served licences, take it there will be special logic to use as licence start date in the future?")
   fun getLicenceStartDate(
     nomisRecord: PrisonerSearchPrisoner,
-    licenceKind: LicenceKind? = null,
-  ): LocalDate? {
-    val lsdKind = licenceKind ?: determineReleaseDateKind(
-      nomisRecord.postRecallReleaseDate,
-      nomisRecord.conditionalReleaseDate,
+    licenceKind: LicenceKind?,
+  ): LocalDate? = when (licenceKind) {
+    HDC -> nomisRecord.homeDetentionCurfewActualDate
+    PRRD -> nomisRecord.calculatePrrdLicenceStartDate()
+    CRD -> calculateCrdLicenceStartDate(
+      nomisRecord,
+      iS91DeterminationService.isIS91Case(nomisRecord),
     )
-
-    return when (lsdKind) {
-      HDC -> nomisRecord.homeDetentionCurfewActualDate
-      PRRD -> nomisRecord.calculatePrrdLicenceStartDate()
-      else -> calculateCrdLicenceStartDate(
-        nomisRecord,
-        iS91DeterminationService.isIS91Case(nomisRecord),
-      )
-    }
+    else -> null
```

Comment:
A null `eligibleKind` (as a result of being made ineligible for CVL) now means LSD will be nulled, making the case `attention needed`. This seemed better than defaulting to using CRD dates.